client/src/components/InsertShorty.js

```diff
@@ -55,7 +68,7 @@ const InsertShorty = ({ onSuccess }) => {
         disabled={loading}
       />
       {error && <div>💀 {error.message}</div>}
```

Comment:
Not sure about the `div` here. Could you leave it out? It's unsemantic but also not used for styling. Or is it?

Reply:
It's a block element and will be displayed in a new line. Without `div`, it would appear at the end of the line. Better styling would be great :) .